### PR TITLE
Changing the APT test logic in install.sh

### PR DESCRIPTION
**Pull request by bureado**
Changing pre-apt test to use ID_LIKE instead of just ID, potentially enabling installation in more Debian-derived distributions. This is related to #355.

### Pull Request Checklist

- [x] Review the [contributing guidelines](https://github.com/Azure/batch-shipyard/blob/master/CONTRIBUTING.md)
- [x] Outstanding issue linked, if applicable
- [x] PR should only be opened against `master` if it includes no features (open against `develop` otherwise)
- [x] Commit messages should conform to [good style practices](https://chris.beams.io/posts/git-commit/)
- [x] PR should pass all checks and have no conflicts

### Description

This was successfully tested in a Debian-derived distro that has an arbitrary ID but where /etc/os-release also had an ID_LIKE pointing to Debian. It should also be tested in Debian and Ubuntu proper (across all supported releases), noting that Ubuntu uses ID_LIKE as well.

### CODE REVIEW

**Comment by edyoung** [comment]
Hi folks, any updates on this one? Currently blocking upgrading Azure Cloud Shell to a newer base image. If this can't be resolved we will have to drop batch-shipyard because we can't stay on current image indefinitely.

**Comment by alfpark** [comment]
@edyoung thanks for following up - is there a proposed drop date?

**Comment by edyoung** [comment]
@alfpark I'm still trying to fix a couple other issues but I would like to release this month or early next.

File: install.sh [hunk 1]
```diff
@@ -157,7 +158,7 @@ echo "Detected OS: $DISTRIB_ID $DISTRIB_RELEASE"
```
Comment (edyoung) [inline install.sh:158]: you might want to echo $DISTRIB_LIKE as well

**Review by edyoung — APPROVED** [review]

**Comment by alfpark** [comment]
`cbld` and `ID_LIKE` will be supported.

**Pull request closed without merging**